cli/Valet/Site.php

```diff
@@ -477,8 +482,11 @@ public function secure($url, $siteConf = null, $caExpireInDays = 368)
 
         $this->files->ensureDirExists($this->nginxPath(), user());
 
-        $this->createCa($caExpireInDays);
-        $this->createCertificate($url, $caExpireInDays);
+        $caExpireInterval = (new \DateInterval(sprintf('P%dY', $caExpireInYears)));
+        $caExpireInDate = (new \DateTime())->diff((new \DateTime())->add($caExpireInterval));
+
+        $this->createCa($caExpireInDate->format('%a'));
```

Comment:
What if we simplify this a bit?

```suggestion
        $caExpireInDate = (new DateTime())->diff(new DateTime("+{$caExpireInYears} years"));
        $this->createCa($caExpireInDate->format('%a'));
```

Reply:
Much better, yes. :+1: